features/multiple_apps.feature

```diff
@@ -20,7 +20,7 @@ Feature: Multiple Applications Test
    #Confirm 2 apps on Login Screen
     Then I go back
     Then I see the app selector button
-    Then I press "Test: List Searching"
+    Then I press view with id "app_selection_spinner"

```

Comment:
No, not that I'm aware of. As far as I know, every place in our tests that requires clicking on a specific item uses 1 of 2 patterns: either pressing a view with a specific ID, or clicking on a specific text value (like what I just replaced here). Of those 2 options, the ID pattern seems slightly better to me since it's less likely to change and guaranteed to be unique (the reason I had to change this just now is because there was another place on the screen with the same "Test: List Searching" text, so the test was failing).